Anoosha

Anoosha-Sehar opened the pull request:
Genome Tracker terms

Anoosha-Sehar committed:
Updated hog meal

changed vegetation-based subtances to plant-based substances.

Anoosha-Sehar committed:
incense powder

Anoosha-Sehar committed:
Added animal by-product

Anoosha-Sehar committed:
Added a referenced term 'cut of beef, mexican'

Moved from scr/ontology/foodon-edit.owl to  foodon_product_import.owl

Anoosha-Sehar committed:
Added 'diezmillo'

Moved from src/ontology/foodon-edit.owl to foodon_product_import.owl

Anoosha-Sehar committed:
Added 'meat (tenderized)'

Anoosha-Sehar committed:
Added 'party wings'

Anoosha-Sehar committed:
Added 'feed stuff'

Anoosha-Sehar committed:
Updated 'fodder (animal feed)'

Changed the parent class from 'food product for animal' to 'feed stuff'.

Anoosha-Sehar committed:
Added 'petite treat'

ddooley commented on src/ontology/imports/foodon_product_import.owl:
```diff
@@ -33599,6 +33605,109 @@ Fishcakes as defined in the Oxford Dictionary of Food and Nutrition are chopped
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00004164">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_00004161"/>
+        <obo:IAO_0000115 xml:lang="en">A powder made from grounding aromatic plants, woods, and resins.</obo:IAO_0000115>
```
grinding rather than grounding.  Also, you could say "and/or" rather than and

ddooley commented on src/ontology/imports/foodon_product_import.owl:
```diff
@@ -33599,6 +33605,109 @@ Fishcakes as defined in the Oxford Dictionary of Food and Nutrition are chopped
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00004169">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_00001180"/>
+        <obo:IAO_0000115 xml:lang="en">The chicken wings which have been prepared.</obo:IAO_0000115>
```
Drop "The".  
I think its better to label this "prepared chicken wing".  Then "party wing" can be a narrow synonym.  As well you can place existing "Buffalo wing" under it. Note chicken wing is defined in singular.

Anoosha-Sehar replied:
Changed the label. And also made 'chicken wing' a parent term in addition to 'prepared food product'.
'chicken wing' & 'prepared food product' (parent terms)
---prepared chicken wing
---------buffalo wing

ddooley commented on src/ontology/imports/foodon_product_import.owl:
```diff
@@ -33599,6 +33605,109 @@ Fishcakes as defined in the Oxford Dictionary of Food and Nutrition are chopped
+        <obo:IAO_0000117 rdf:resource="https://orcid.org/0000-0001-5275-8866"/>
+        <dc:date rdf:datatype="http://www.w3.org/2001/XMLSchema#dateTime">2022-01-05T11:00:23Z</dc:date>
+        <oboInOwl:hasExactSynonym xml:lang="en">mini-bite treat</oboInOwl:hasExactSynonym>
+        <rdfs:label xml:lang="en">petite treat</rdfs:label>
```
I lookup "petite treat" on the web and find references to small pastries etc.  So I think "small pet treat" would be a better label (relative to size of animal eating it of course).  I think petite treat is a brand name. But if you want, make it a narrow synonym, and same with "mini-bite treat".  And can we make the definitions as Aristotelian as possible? So, a pet treat which is small in size".  Treats themselves may not be designed to meet [nutritional] needs of an animal, though pleasure is at play!

Anoosha-Sehar replied:
Totally agree. Few brands mentioned the use of these small-bite treats for supplemental feeding, so got confused with that. It's better to change the label and add it as a narrow synonym instead.

ddooley commented on src/ontology/imports/foodon_product_import.owl:
```diff
@@ -33599,6 +33605,109 @@ Fishcakes as defined in the Oxford Dictionary of Food and Nutrition are chopped
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00004170">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_03309997"/>
+        <obo:IAO_0000115 xml:lang="en">Any foodstuff used to feed livestock.</obo:IAO_0000115>
```
I can't quite recall our discussion about this, but there was the distinction between "feed" like foraging crops as meant for mainly herbaceous creatures; whereas a more general term for any kind of feedstuff for animals was needed.  But FoodOn already has ENVO term "animal feed".  Should we ask ENVO to take over this term, and add the exact synonyms? Or ask them to add the synonyms?

Anoosha-Sehar replied:
I think in the very end we discussed to organize it somewhat like this (which I have in my notes). 
--feed stuff
---animal feed (ENVO term?)
-----fodder (animal feed)
FoodOn has '`foodstuff`' as a synonym of '`food material`'. 
And definition of 'animal feed' says that '`Food material` which can be used to meet nutritional requirements of animals, particularly livestock such as cattle, goats, sheep, horses, chickens and pigs.
so does it makes sense to have something like this? 
-food material
--feed stuff
---animal feed (ENVO term)
-----fodder

Anoosha-Sehar replied:
I remember that you also proposed a hierarchy in zoom chat (14Dec), but unfortunately Zoom didn't save my chat. Do you still have that?

ddooley replied:
I think ENVO "animal feed" is the broadest term under which all food materials intended for animals is placed. Wikipedia backs this up: https://en.wikipedia.org/wiki/Animal_feed   
Given FoodOn's mandate, I think "animal feed" could be converted to a new FOODON term.  Can you put in a request for that to ENVO to see what they say?  Then we can have that term under "food product for animal".  And then add feedstuff and feed stuff as exact synonyms.  I'm seeing this now from the diagram in http://www.ndvsu.org/images/StudyMaterials/Nutrition/Classification-of-feedstuffs.pdf 

Then FoodOn "fodder (animal feed)" can be placed under "animal feed".

ddooley replied:
Actually, I'm thinking feedstuff could be the new FoodOn term to replace "animal feed".  The issue is that "animal feed" sounds like it could cover pet food, but really feedstuff is meant for agricultural production - which is what ENVO meant, rather than pet food.  "Feedstuff" is more consistent with ENVO definition.

Anoosha-Sehar replied:
Right! Then 'animal feed' looks more appropriate to me than 'feedstuff' as a new FoodOn term to accommodate any kind of animal feedstuff. I will make a request to ENVO, to see if we can convert 'animal feed' to a new FoodOn term. And then we can place foodstuff like 'fodder' underneath. 'feedstuff', 'feed stuff', 'feed for animals' can be exact synonyms.

Removing the term 'feedstuff' for now.

ddooley replied:
I think ENVO "animal feed" -> FoodOn "feedstuff"
Then we can have
"food for animals"
   feedstuff (food for agricultural animals)
        fodder
        etc.
   pet food
       cat food
       dog food
       etc.

Anoosha-Sehar replied:
Just to confirm, from 'food for animals', we mean 'food product for animals', right? 
 like this:
-food product for animals
 --feedstuff (food for agricultural animals)
   ----fodder (animal feed)
 --pet food
 --cat food
 --dog food

ddooley replied:
Right!

ddooley replied:
p.s. cat food and dog food would be indented under pet food (as they are now).

Anoosha-Sehar replied:
Exactly!

Anoosha-Sehar committed:
Updated 'incense powder'

Anoosha-Sehar committed:
Added 'prepared chicken wing'

Changed the label from 'party wing' to 'prepared chicken wing.
Also added 'chicken wing' as an other parent class along with 'prepared food product'.
Added Narrow synonym 'party wing'

Anoosha-Sehar committed:
Updated 'buffalo wing'

Added under 'prepared chicken wing'

Anoosha-Sehar committed:
changed 'petite treat' to 'small pet treat'

Anoosha-Sehar committed:
Deleted 'feed stuff'

Reversed the changes made to 'fodder (animal feed)'

ddooley commented on src/ontology/imports/foodon_product_import.owl:
```diff
@@ -33599,6 +33605,98 @@ Fishcakes as defined in the Oxford Dictionary of Food and Nutrition are chopped
+        <obo:IAO_0000115 xml:lang="en">A pet treat which is small in size.</obo:IAO_0000115>
+        <obo:IAO_0000117 rdf:resource="https://orcid.org/0000-0001-5275-8866"/>
+        <dc:date rdf:datatype="http://www.w3.org/2001/XMLSchema#dateTime">2022-01-05T11:00:23Z</dc:date>
+        <oboInOwl:hasNarrowSynonym xml:lang="en">mini-bite treat</oboInOwl:hasNarrowSynonym>
```
Ugh, sorry - I think I made a mistake in saying mini-bite treat and petite treat are narrow synonyms. I think they are exact synonyms (where both have hidden context of pet treats).

Anoosha-Sehar committed:
Updated synonyms
